pkg/config/config.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/adrg/xdg"
+)
+
+type Config struct {
+
+	// Home directory for the avm configuration.
+	homeDir string
```

Comment:
This should be `configDir`, right?

Reply:
However, I would change one of the follow-up commits.